packages/pn-personafisica-webapp/src/pages/Profile.page.tsx

```diff
@@ -93,7 +93,6 @@ const Profile = () => {
 
         <Grid item lg={8} xs={12}>
           <Alert
-            severity="info"
             aria-label="contacts-redirect"
```

Comment:
This aria-label can be removed?

Reply:
Yes, but I have replaced `aria-label` with `data-testid` because it is used on tests